### ISSUE TITLE
Mac bundle is successful exported but it closes when trying to open

### ISSUE THREAD

**Issue by ivanhake** [fvarrui/JavaPackager#53]
Hi there, first of all, congrats for the project is really cool!! 

I'm trying to generate the mac bundle for a java project, I follow the configuration in the documentation, it builds perfectly and all the files are generated. However, when I try to run the .app, the application closes itself. Without any error message. 

I have the following XML plugin configuration.  

```xml
        <plugin>
              <groupId>io.github.fvarrui</groupId>
	       <artifactId>javapackager</artifactId>
		<version>1.1.0</version>
		<executions>
		        <execution>
			        <phase>package</phase>
			        <goals>
			                <goal>package</goal>
			        </goals>
			        <configuration>
			                <!-- mandatory -->
			                <mainClass>com.navi.agenslib.AgensLib</mainClass>
			                <!-- optional -->
			                <bundleJre>true</bundleJre>
			                <generateInstaller>true</generateInstaller>        
			                <administratorRequired>true</administratorRequired>
			                <platform>mac</platform>
			                <iconFile>assets/mac/AgensLib.icns</iconFile>
			                <macConfig>
			               		<!-- general properties -->
			               		<generateDmg>true</generateDmg>
						<generatePkg>true</generatePkg>
						<backgroundImage>assets/mac/background.png</backgroundImage>
						<windowWidth>540</windowWidth>
						<windowHeight>360</windowHeight>
						<iconSize>128</iconSize>
						<textSize>16</textSize>
						<iconX>52</iconX>
						<iconY>116</iconY>
						<appsLinkIconX>360</appsLinkIconX>
						<appsLinkIconY>116</appsLinkIconY>
						<volumeIcon>assets/mac/AgensLib.icns</volumeIcon>
						<volumeName>${displayName}</volumeName>
				        </macConfig>
			        </configuration>
			</execution>
                </executions>
	</plugin>
```

The final console output after the maven package command line, is the following:

``` sh
[info]     Generating Mac OS X installer package ...
[info]         Executing command: /bin/sh -c cd '/Users/van/Documents/AgensLib_Desktop/.' && 'pkgbuild' --install-location /Applications --component /Users/van/Documents/AgensLib_Desktop/target/AgensLib/AgensLib.app /Users/van/Documents/AgensLib_Desktop/target/AgensLib_0.0.1.pkg
[info]         pkgbuild: Adding component at /Users/van/Documents/AgensLib_Desktop/target/AgensLib/AgensLib.app
[info]         pkgbuild: Wrote package to /Users/van/Documents/AgensLib_Desktop/target/AgensLib_0.0.1.pkg
[info]     Mac OS X installer package generated in /Users/van/Documents/AgensLib_Desktop/target/AgensLib_0.0.1.pkg!
[info]     
[info] Installers generated! [/Users/van/Documents/AgensLib_Desktop/target/AgensLib_0.0.1.dmg, /Users/van/Documents/AgensLib_Desktop/target/AgensLib_0.0.1.pkg]
[info] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  28.972 s
[INFO] Finished at: 2020-08-29T00:49:50-05:00
[INFO] ------------------------------------------------------------------------
```

**Comment by fvarrui** [comment]
Hi @ivanhake!
Thanks to you for using JavaPackager.

I think I was able to reproduce your issue and find out what's wrong by running the generated ".app" from the commandline:

1. Open terminal.
2. ```cd /Users/van/Documents/AgensLib_Desktop/target/AgensLib/Contents/MacOS/```
3. ```./startup``` ... with my testing app it shows a message like this:

![image](https://user-images.githubusercontent.com/4354486/91632004-b60f8e00-e9d5-11ea-86eb-152468ff6479.png)

This happens because we are trying to run our apps with administrator privileges
```xml
<administratorPrivileges>true<administratorPrivileges/>
```
and there's an error in the generated `startup` script in this case.
![image](https://user-images.githubusercontent.com/4354486/91632078-464dd300-e9d6-11ea-9416-67bdc47e5f83.png)
`$SCRIPTPATH` variable is not being interpreted by the shell, as it's using single quotes instead of double quotes ... what a mess!  😄 

When I replace single quotes with double quotes:
![image](https://user-images.githubusercontent.com/4354486/91632192-2834a280-e9d7-11ea-9234-3bf3e6a45d6d.png)
it runs as expected:
![image](https://user-images.githubusercontent.com/4354486/91632134-b9574980-e9d6-11ea-8515-cb535d11e3be.png)

Sorry, I probably introduced this error to deal with paths contaning whitespaces.

I've just fixed in **1.2.0-SNAPSHOT**, which requires a [manual installation](https://github.com/fvarrui/JavaPackager#how-to-build-and-install-the-plugin) in your local repo till I release it to Maven Central as 1.2.0.

**Comment by ivanhake** [comment]
It works perfectly with that change. 

Thank you so much!
